### PR TITLE
added room message field

### PR DESCRIPTION
**Pull request by EliAndrewC**
An admin can fill out a new field on the Create Room (or Edit Room) form which will be added to the automated email for that room once the room is locked in.

This requires a database change which I've already done manually on the staging and production servers after testing it locally on my own box:

``` sql
ALTER TABLE room ADD COLUMN message TEXT NOT NULL DEFAULT '';
```

### CODE REVIEW

File: hotel/site_sections/hotel_assignments.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def ordered(self, out, session):
                 writerow(a, a.hotel_requests)
 
     @csv_file
-    def hilton_mark_center(self, out, session):
+    def passkey(self, out, session):
```
Comment (EliAndrewC) [inline hotel/site_sections/hotel_assignments.py:142]: Unrelated fix for #7 I made while was in here anyway.

**Comment by EliAndrewC** [comment]
Merging now since this is a small change I've already tested which Stops could use ASAP.

**Comment by binary1230** [comment]
+1
